src/services/storage-service.js

```diff
@@ -228,8 +285,8 @@ export default class StorageService {
     }
 
     const room = await this.roomStore.getRoomById(doc.roomId);
-    const isRoomOwner = user._id === room.owner;
-    const roomOwner = isRoomOwner ? user : await this.userStore.getUserById(room.owner);
+    const isUserRoomOwner = user._id === room.owner;
```

Comment:
This variable name reminds me of something 🤔

Reply:
I copy/pasted it from `updateUserLastLoggedInOn`. You're right, it's way too late at this point to validate it :thinking:  Removed (in both places.